src/rauc-hawkbit-updater.c

```diff
@@ -62,26 +62,30 @@ static GSourceFunc notify_hawkbit_install_complete;
 
 
 /**
- * @brief RAUC callback when new progress.
+ * @brief Callback for install thread, consumes RAUC progress messages, logs them and passes them
+ * on to notify_hawkbit_install_progress().
+ *
+ * @param[in] data install_context pointer allowing access to received status messages
+ * @return G_SOURCE_REMOVE is always returned
```

Comment:
Semantically, this is a `GSourceFunc()`, maybe this is worth being added to the documentation?

Reply:
Yes, I've added this info to all `GSourceFunc`s now.